cscs-checks/apps/quantumespresso/quantumespresso_check.py

```diff
@@ -39,17 +39,43 @@ class QuantumESPRESSOCpuCheck(QuantumESPRESSOCheck):
     def __init__(self, scale, variant):
         super().__init__()
         self.descr = f'QuantumESPRESSO CPU check (version: {scale}, {variant})'
-        self.valid_systems = ['daint:mc']
+        self.valid_systems = ['daint:mc', 'eiger:mc']
         self.modules = ['QuantumESPRESSO']
         if scale == 'small':
             self.valid_systems += ['dom:mc']
-            self.num_tasks = 216
             energy_reference = -11427.09017218
+            if self.current_system.name in ['daint', 'dom']:
+                self.num_tasks = 216
+                self.num_tasks_per_node = 36
+            elif self.current_system.name == 'eiger':
+                self.num_tasks = 96
+                self.num_tasks_per_node = 16
+                self.num_cpus_per_task = 16
+                self.num_tasks_per_core = 1
+                self.use_multithreading = False
+                self.variables = {
+                    'MPICH_OFI_STARTUP_CONNECT': '1',
+                    'OMP_NUM_THREADS': '8',
+                    'OMP_PLACES': 'cores',
+                    'OMP_PROC_BIND': 'close'
+                }
         else:
-            self.num_tasks = 576
             energy_reference = -11427.09017152
-
-        self.num_tasks_per_node = 36
+            if self.current_system.name in ['daint', 'dom']:
```

Comment:
```suggestion
            if self.current_system.name in ['daint']:
```
It doesn't hurt to keep it, but we don't run `large` on `dom`

Reply:
Thanks @ekouts